Tests/fixtures/podcasting.yml

```diff
@@ -0,0 +1,9536 @@
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'SoundOn/2.2.2 (iOS)'
+  os:
+    name: iOS
+    version: ""
+    platform: ""
+  client:
+    type: mobile app
+    name: SoundOn
+    version: "2.2.2"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'SoundOn/656 CFNetwork/1240.0.4 Darwin/20.6.0'
+  os:
+    name: iOS
+    version: "14.7"
+    platform: ""
+  client:
+    type: mobile app
+    name: SoundOn
+    version: "656"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'SoundWaves-0.160.2'
+  os: [ ]
+  client:
+    type: mobile app
+    name: SoundWaves
+    version: "0.160.2"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2710.89 Safari/537.36 Spreaker/1.0'
+  os:
+    name: Mac
+    version: "10.9.5"
+    platform: ""
+  client:
+    type: mobile app
+    name: Spreaker
+    version: "1.0"
+  device:
+    type: desktop
+    brand: Apple
+    model: ''
+  os_family: Mac
+  browser_family: Unknown
+
+-
+  user_agent: 'Stitcher Demo/4.8.0 (Linux;Android 11) ExoPlayerLib/2.10.7'
+  os:
+    name: Android
+    version: "11"
+    platform: ""
+  client:
+    type: mobile app
+    name: Stitcher
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'Stitcher/28420 CFNetwork/609.1.4 Darwin/13.0.0'
+  os:
+    name: iOS
+    version: "6.1"
+    platform: ""
+  client:
+    type: mobile app
+    name: Stitcher
+    version: ""
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Stitcher/iOS dWZpZA'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Stitcher
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'StoryShots/1.9.72 (Linux;Android 9) ExoPlayerLib/2.11.4'
+  os:
+    name: Android
+    version: "9"
+    platform: ""
+  client:
+    type: mobile app
+    name: StoryShots
+    version: "1.9.72"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'Swinsian/472 CFNetwork/978.0.7 Darwin/18.7.0 (x86_64)'
+  os:
+    name: Mac
+    version: "10.14"
+    platform: x64
+  client:
+    type: mobile app
+    name: Swinsian
+    version: "472"
+  device:
+    type: desktop
+    brand: Apple
+    model: ''
+  os_family: Mac
+  browser_family: Unknown
+
+-
+  user_agent: 'ThePodcastApp/6.49.0 (iPhone; iOS 16.5.1; ) player (build 6775;  https://podcast.app/)'
+  os:
+    name: iOS
+    version: "16.5.1"
+    platform: ""
+  client:
+    type: mobile app
+    name: The Podcast App
```

Comment:
Based on https://apps.apple.com/us/app/podcast-app/id1199070742 and https://podcast.app/ they do not use "The" in the name anymore